src/modules/sales/sales-action.js

```diff
@@ -181,34 +193,42 @@ export const updateCommentAction = async (data, index) => {
   });
 
   const influencerCollection = DB.collection('influencersSalesProducts');
+  let influencerRefGet;
   let influencer;
   let idInfluencer;
 
-  await influencerCollection
-    .where('saleId', '==', ref.id)
-    .get()
-    .then((data) => {
-      data.forEach((doc) => {
-        idInfluencer = doc.id;
-        influencer = doc.data();
-        if (ref.comment) {
-          influencer.comment = ref.comment;
-          influencer.orderStatus = sale.orderStatus;
-        }
-      });
-    })
-    .catch((e) => {
-      Flux.dispatchEvent(COMMENT_ERROR, new Error(e));
-      console.log(e);
-    });
+  try {
+    influencerRefGet = await influencerCollection
+      .where('saleId', '==', ref.id)
+      .get();
+  } catch (err) {
+    return Flux.dispatch(COMMENT_ERROR, new Error(err));
+  }
+
+  influencerRefGet.forEach((element) => {
+    if (element.data().saleId === ref.id) {
```

Comment:
```suggestion
    if (element.data().saleId === ref.id) {
```
esto no es necesario